Event-Aggregator/Views/Details/Index.cshtml

```diff
@@ -45,45 +34,43 @@
                         <div class="row">
                             <div class="glyphicon glyphicon-map-marker col-md-4" style="font-size: 40px"></div>
                             <div class="col-md-8">
-                                <p>@EventLocalization</p>
+                                <p>@Model.Localization</p>
                             </div>
                         </div>
                     </div>
                 </div>
             </div>
             <div id="Description" class="col-md-12">
-                <p>@EventDesc</p>
+                <p>@Model.Description</p>
             </div>
 
         </div>
         <div class="col-md-6 col-sm-12">
             <div id="map"></div>
             <script>
                 function initMap() {
-                    let address = "@Html.Raw(item.Localization)";
+                    let address = "@Html.Raw(Model.Localization)";
                     var geocoder = new google.maps.Geocoder();
                     var zoom = { zoom: 15 };
                     var map = new google.maps.Map(document.getElementById('map'), zoom);
 
                     geocoder.geocode({ 'address': address }, function (results, status) {
                         if (status == 'OK') {
-
-                            /*, { center: results[0].geometry.location, zoom: 15 })*/
                             map.setCenter(results[0].geometry.location);
                             var marker = new google.maps.Marker({
                                 map: map,
                                 position: results[0].geometry.location
                             });
                         } else {
-                            alert('Geocode was not successful for the following reason: ' + status);
+                            console.log('Geocode was not successful for the following reason: ' + status);
                         }
                     });
                 }
             </script>
-            <script src="https://maps.googleapis.com/maps/api/js?key=[nohehe]&callback=initMap" async defer></script>
+            <script src="https://maps.googleapis.com/maps/api/js?key=@ViewBag.ApiKey&callback=initMap" async defer></script>
```

Comment:
Consider loading the script in the header using defer itself 😄

Reply:
Done sir.